app/boards/arm/cyber60/cyber60.dts

```diff
@@ -18,20 +18,16 @@
 		zmk,kscan = &kscan0;
 		zmk,matrix_transform = &default_transform;
 	};
-/*
-	vbat-sense-en {
-		label = "VBAT_SENSE_EN";
-		control-gpios = <&gpio1 15 GPIO_ACTIVE_HIGH>;
-	};
-*/
+
 	vbatt {
 		compatible = "zmk,battery-voltage-divider";
 		label = "BATTERY";
 		io-channels = <&adc 1>;
 		output-ohms = <100000>;
 		full-ohms = <(100000 + 100000)>;
+		power-gpios = <&gpio1 15 1>; // the 1 indicates active high?
```

Comment:
`GPIO_ACTIVE_HIGH` is equivalent to 0 and `GPIO_ACTIVE_LOW` is equivalent to 1. You should be able to keep `GPIO_ACTIVE_HIGH` in here instead of the number.

Reply:
Of course, thanks!